app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -128,7 +128,7 @@ def inspect_work
     def manifest
       headers['Access-Control-Allow-Origin'] = '*'
 
-      json = iiif_manifest_builder.manifest_for(presenter: presenter)
+      json = iiif_manifest_builder.manifest_for(presenter: iiif_manifest_presenter)
```

Comment:
I like using a specific presenter instead of the big heavy work presenter. Only "present" the smallest necessary object to complete the request.

Reply:
i'd like to do more of this, in general.

for me, the power of the presenter pattern is that i can decorate an object for a specific purpose. making presenters monolithic seems like an anti-pattern for me